packages/ia-components/sandbox/theatres/with-youtube-spotify/audio-with-youtube-spotify-main.jsx

```diff
@@ -68,7 +68,7 @@ class AudioPlayerWithYoutubeSpotify extends Component {
       albumData,
       tracklistToShow: [],
       channelToPlay: 'archive',
-      trackSelected: 1, /* 0 = album */
+      trackSelected: null, /* 0 = album */
```

Comment:
SUGGESTION: You note what value `0` means. What about its default value of `null`?

Reply:
`null` signifies that nothing has been chosen yet by user.  This update is to integrate in fix for playing an item on first click.